sass/styles.scss

```diff
@@ -1 +1,547 @@
-p {color:red;}
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:300,400,700|Roboto:100,300,400,500,700');
+
+/*variables*/
+$color-accent: #cc071e;
+$color-body: #353535;
+$body-font: Roboto, sans-serif;
+
+// $link: #000000;
+// $visit: #cc071e;
+// $hover: #cc071e;
+// $active: #000000;
+
+/*mixins*/
+@mixin link($link, $hover, $active) {
```

Comment:
Nice mixin idea here!

Reply:
But now seeing its use it might be a bit fabricated. A better use for mixin is for code you repeat frequently. I'll look for such instances and flag them.